resources/views/vendors/vendors/individualVendor.blade.php

```diff
@@ -34,73 +35,129 @@
                     <label for="#">Email</label>
-                    <input type="email" class="form-control" value="pocketdevs@example.com.ph">
+                    <input type="email" class="form-control" name="email" value="{{$vendor->email}}">
                 </div>
                 <div class="col">
                     <label for="#">Website</label>
-                    <input type="text" class="form-control" value="pocketdevs.ph">
+                    <input type="text" class="form-control" name="website" value="{{$vendor->website}}">
+                </div>
+            </div>    
+            <div class="row form-group mt-4">
+                <label for="label" class="col-md-4 col-lg-2 col-form-label">Label:</label>
+                <select name="label" class="form-control col-md-2 col-lg-2">
+                    <option value="New" {{ $vendor->label == 'New' ? 'selected' : '' }}>New</option>
+                    <option value="VIP"  {{ $vendor->label == 'VIP' ? 'selected' : '' }}>VIP</option>
+                    <option value="ISP"  {{ $vendor->label == 'ISP' ? 'selected' : '' }}>ISP</option>
+                </select>
+                <label for="image" class="mx-3 col-form-label">Image:</label>
+                <div class="input-group col-md-4 col-lg-4">
+                    <div class="custom-file">
+                    <input type="file" class="custom-file-input" id="inputGroupFile03" name="image">
+                    <label class="custom-file-label" for="inputGroupFile03">Choose file</label>
+                    </div>
+                </div>
+                <label for="is_active" class="col-form-label">Status:</label>
+                <div class="form-check ml-5">
+                    <div class="row">
+                        <input class="form-check-input" type="radio" name="is_active" id="r_paymentType_credit" value="Yes" {{ $vendor->is_active == 'Yes' ? 'checked' : '' }}>
+                        <label class="form-check-label" for="r_paymentType_credit">
+                        Active
+                        </label>
+                    </div>
+                    <div class="row">
+                        <input class="form-check-input" type="radio" name="is_active" id="r_paymentType_cash" value="No" {{ $vendor->is_active == 'No' ? 'checked' : '' }}>
+                        <label class="form-check-label" for="r_paymentType_cash">
+                        Inactive
+                        </label>
+                    </div>
                 </div>
             </div>
             <div class="row d-flex justify-content-end mt-3">
                 <div>
-                    <button class="btn btn-secondary mx-1">Edit</button>
-                    <button class="btn btn-danger mr-3">Delete</button>
+                    <button class="btn btn-secondary mx-1" type="submit">Edit</button>
+                    <button class="btn btn-danger mr-3 " type="button" data-toggle="modal" data-target="#deleteModal">Delete</button>
```

Comment:
Can you move the `delete` button to the very left while leaving the `edit` button alone on the right? It may be mistakenly clicked by users assuming it will update the vendor.

Also, I think `Update Vendor` is the appropriate term to use in the `Edit` button for better clarity to users on its purpose. Lastly, I think it should redirect back to the vendors page.